app/views/components/itemDetailSection.scala.html

```diff
@@ -0,0 +1,42 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.DetailsSelectItemHelper
+@import models.response.emcsTfe.MovementItem
+@import models.response.referenceData.CnCodeInformation
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(
+    govukDetails : GovukDetails,
+    govukTable: GovukTable,
+    helper: DetailsSelectItemHelper
+)
+
+@(summary: String, item: MovementItem, cnCodeInformation: CnCodeInformation)(implicit messages: Messages)
+
+@govukDetails(Details(
+summary = Text(messages(summary, item.itemUniqueReference)),
+content = HtmlContent(
+govukTable(Table(
+rows = helper.constructItemSummaryRows(item, cnCodeInformation),
+firstCellIsHeader = true
+))
+)
+))
```

Comment:
```suggestion
@govukDetails(Details(
    summary = Text(messages(summary, item.itemUniqueReference)),
    content = HtmlContent(
        govukTable(Table(
            rows = helper.constructItemSummaryRows(item, cnCodeInformation),
            firstCellIsHeader = true
        ))
    )
))
```
Just formatting this a bit more nicely